lblod/frontend-subsidiepunt
# Feature: export to pdf

## ID

DGS-306

## Description

Generate pdf for subsidy forms

## Type of change


## How to test

Open subsidies and click 'Download huidige stap (PDF)' to verify working and valid pdf's of the subsidy step.
